ppc64le_build.bash

```diff
@@ -1,22 +1,50 @@
 echo "#####################"
 
 set -eux
 
-#apt-get update
-#apt-get install -y build-essential clang git vim cmake python libcups2-dev pkg-config libnss3-dev libssl-dev libglib2.0-dev libgnome-keyring-dev libpango1.0-dev libdbus-1-dev libatk1.0-dev libatk-bridge2.0-dev libgtk-3-dev libkrb5-dev libpulse-dev libxss-dev re2c subversion curl libasound2-dev libpci-dev mesa-common-dev gperf bison uuid-dev clang-format libatspi2.0-dev libnotify-dev libgconf2-dev libcap-dev libxtst-dev libxss1 python-dbusmock openjdk-8-jre ninja-build clang-format wget
+. /etc/os-release
+OS=$NAME
+OSVER=$VERSION_ID
+
+if [[ $OS == "Ubuntu" ]] && [[ $OSVER == "16.04" ]]
+then
+    apt-get update -y
+    apt-get install -y build-essential clang git vim cmake python bzip2 tar pkg-config libcups2-dev pkg-config libnss3-dev libssl-dev libglib2.0-dev libgnome-keyring-dev libpango1.0-dev libdbus-1-dev libatk1.0-dev libatk-bridge2.0-dev libgtk-3-dev libkrb5-dev libpulse-dev libxss-dev re2c subversion curl libasound2-dev libpci-dev mesa-common-dev gperf bison uuid-dev clang-format libatspi2.0-dev libnotify-dev libgconf2-dev libcap-dev libxtst-dev libxss1 python-dbusmock openjdk-8-jre clang-format wget libnotify-dev
+
+    apt-get install -y software-properties-common
+    add-apt-repository ppa:deadsnakes/ppa -y
+    apt-get update -y
+    apt-get install -y python3.7
+    ln -sf /usr/bin/python3.7 /usr/bin/python3
+
+    git clone https://github.com/ninja-build/ninja
+    cd ninja
+    git checkout v1.7.2
+    ./configure.py --bootstrap
+    export PATH=$(pwd):$PATH
+    cd ..
+elif [[ $OS == "Fedora" ]] && [[ $OSVER == "30" ]]
+then
+    # Install dependencies
+    dnf install -y git wget python bzip2 tar pkgconfig atk-devel alsa-lib-devel bison binutils brlapi-devel bluez-libs-devel bzip2-devel cairo-devel cups-devel dbus-devel dbus-glib-devel expat-devel fontconfig-devel freetype-devel gcc-c++ glib2-devel glibc gperf gtk3-devel java-1.*.0-openjdk-devel libatomic libcap-devel libffi-devel libgcc libgnome-keyring-devel libjpeg-devel libstdc++ libX11-devel libXScrnSaver-devel libXtst-devel libxkbcommon-x11-devel ncurses-compat-libs nspr-devel nss-devel pam-devel pango-devel pciutils-devel pulseaudio-libs-devel zlib httpd mod_ssl php php-cli python-psutil wdiff xorg-x11-server-Xvfb clang vim cmake pkg-config krb5-devel re2c subversion curl libuuid-devel ninja-build libva-devel ccache xz patch diffutils findutils coreutils @development-tools elfutils rpm-devel rpm rpm-build fakeroot libgbm-devel libXcomposite gdk-pixbuf2-devel libnotify-devel
+else
+    echo "Unsupported OS, please consider installing/building respective dependencies!"
+    exit 0
+fi
 
 VERSION=v10.20.1
 DISTRO=linux-ppc64le
 
 wget "https://nodejs.org/dist/$VERSION/node-$VERSION-$DISTRO.tar.xz"
 tar -xJvf node-$VERSION-$DISTRO.tar.xz
 PATH=$(pwd)/node-$VERSION-$DISTRO/bin:$PATH
+ln -sf $(pwd)/node-$VERSION-$DISTRO/bin/node /usr/bin/node
```

Comment:
You should not create a link in a system directory, manipulating the PATH variable and patching build scripts if they reference absolute paths should be sufficient.

We do not want the build script to require being run as root

Reply:
OK, I'll try to fix the reference to absolute path '/usr/bin/node' in the build. Same thing will apply for the python3.7 ubuntu link too. Will work on that one too.

Reply:
Didnt see the Python one, yes please do!

Reply:
Completed the necessary changes. Awaiting approval for contributing to https://github.com/leo-lb/chromium_power (specifically to https://github.com/leo-lb/chromium_power/blob/master/third_party/0001-DONTMERGE-third_party-node-Use-system-nodejs-binary.patch) w.r.t node path.

Reply:
Please review https://github.com/leo-lb/chromium_power/pull/3 for the node path fix.